src/Symfony/Component/HttpKernel/DependencyInjection/ServicesResetter.php

```diff
@@ -20,8 +20,6 @@
  *
  * @author Alexander M. Turek <me@derrabus.de>
  * @author Nicolas Grekas <p@tchwork.com>
- *
- * @internal
  */
 class ServicesResetter implements ResetInterface
```

Comment:
Should it be @final now and real final in 8.0 then?

Reply:
I agree.

Reply:
updated 👍🏼

Reply:
Since the class was already `@internal`, why not changing to a real `final` now? No one is supposed to use it in their code.

Reply:
"suppose" is the important piece here ;) But I agree with you.